cypress/e2e/tests/migration/applicationinventory/applications/language_discovery.test.ts

```diff
@@ -15,16 +15,26 @@ limitations under the License.
 */
 /// <reference types="cypress" />
 
-import { login, getRandomApplicationData, sidedrawerTab } from "../../../../../utils/utils";
+import {
+    login,
+    getRandomApplicationData,
+    sidedrawerTab,
+    deleteByList,
+} from "../../../../../utils/utils";
 import { Application } from "../../../../models/migration/applicationinventory/application";
 import { SEC } from "../../../../types/constants";
 import { labelTagText } from "../../../../views/applicationinventory.view";
 
-let application: Application;
+let applicationList: Application[];
```

Comment:
Line 33 could be removed and line 28 could be replaced with:
let applicationList: Application[] = [];

Reply:
done